feature-libs/checkout/base/components/checkout-billing-address/checkout-billing-address-form.component.html

```diff
@@ -0,0 +1,211 @@
+<!-- BILLING -->
```

Comment:
Please make sure to test this feature from a customer perspective:
<img width="624" alt="image" src="https://github.com/SAP/spartacus/assets/34665674/66d08be9-cd29-48ec-9067-8b7fdaa3076b">

Reply:
done